### PR TITLE
Implement session stats

### PR DESCRIPTION
**Pull request by kasia251011**

### CODE REVIEW

**Review by Ewangelina — APPROVED** [review]

File: .idea/.gitignore
Comment (kasia251011) [inline .idea/.gitignore:1]: delete this file

File: .idea/ZPI2022_IO3_Tech_Titans.iml
Comment (kasia251011) [inline .idea/ZPI2022_IO3_Tech_Titans.iml:1]: delete this file

File: .idea/inspectionProfiles/Project_Default.xml
Comment (kasia251011) [inline .idea/inspectionProfiles/Project_Default.xml:1]: thelete whole .idea folder

File: src/components/SessionStats/SelectCurrency/SelectCurrency.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  const inputElement = screen.getByRole('button');
+  expect(inputElement).toBeInTheDocument;
+
+  const aud = screen.findByText('AUD');
+  expect(aud).toBeInTheDocument;
+  const BGN = screen.findByText('BGN');
+  expect(BGN).toBeInTheDocument;
+  const ILS = screen.findByText('ILS');
+  expect(ILS).toBeInTheDocument;
```
Comment (kasia251011) [inline src/components/SessionStats/SelectCurrency/SelectCurrency.spec.tsx:22]: As I said earlier we cannoct depend on api in our tests. There is a chance that api will not retur these currencies, but others, so in this situatin test should fail?

File: src/components/SessionStats/SelectCurrency/SelectCurrency.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  userEvent.click(inputElement);
+
+  const menuElement = screen.findByRole('listbox');
+  expect(menuElement).toBeInTheDocument;
```
Comment (kasia251011) [inline src/components/SessionStats/SelectCurrency/SelectCurrency.spec.tsx:27]: We dont have to test external libraries that we are using. Mui library itself tests if select component after click contains listbox, and tests a lot of other situations...

**Pull request merged**